src/book/02-system/03-tools/06-natnet.mdx

```diff
@@ -0,0 +1,101 @@
+
+1. Ensure that the NatNet role has been installed on the robot.
+
+2. [SSH into the robot](guides/main/getting-started#running-on-a-real-robot).
+
+3. `cd config`
+
+4. `nano DataLogging.yaml`
+
+5. You will need to add message.input.MotionCapture and set its value to true. Additionally, set to true any of the sub messages that you want to use such as ".RigidBody". If you want "output.CompressedImage" or "input.sensors" logged, you can set them to true as well.
+
+6. To exit nano, press Ctrl+X, then 'y', then hit enter.
+
+7. Run Natnet using `./b run natnet`
+
+8. After your program has finished, check that a new nbs file has been created in "log/<your_program's_name>/". The filename should be the date and timestamp when the file was run.
+
+9. Exit out of the ssh of the robot.
+
+10. We now need to copy the nbs file off the robot. Use "scp nubots@10.1.1.<robot_id#>:/home/nubots/log/<your_program_name>/<name_of_nbs_file> ./log/"
+
+11. You now need to extract the information you want from the nbs file. We have a number of .nbs extraction tools and is recommended that you use `./b nbs extract_images <path>` for extracting images from the nbs file and `./b nbs json <path>` to extract the information into JSON format.
+
+</details>
+
+<details>
+<summary>From your computer</summary>
+
+This method may be useful to determine if Motive is outputting packets directly or debugging depacketization without the need of a robot.
+
+1. Ensure that the NatNet role has been installed locally.
+
+2. Ensure that the option `message.input.MotionCapture` in DataLogging.yaml is set to true.
+
+3. Run Natnet using `./b run natnet`
+
+4. After your program has finished, check that a new nbs file has been created in "log/<your_program's_name>/". The filename should be the date and timestamp when the file was run.
+
+5. You now need to extract the information you want from the nbs file. We have a few nbs decoders that have already been written for specific purposes. If one of those does not suit your needs, you will need to either modify an existing nbs extraction file, or write your own. The location of these nbs files is "nubots/tools/nbs_tools/". For example purposes, let's assume you want to extract images and Motion Capture data. To run this program, make sure you are in the nubots home directory, then you would use: "./b nbs extract_images_MoCap ./log/<file_name> --output ./log/<your_extraction_folder>"
+
+</details>
+
+<Alert type='info'>
+NatNet may not always pick up motive packets immediately. This may be dependent of the amount of traffic on your local network or how it is configured. Best Practice to ensure the least flakeyness is to start streaming packets on motive significantly before you run NatNet and if NatNet still doesn't work, try stopping and starting NatNet a few times. If this still doesn't work, try restarting motive streaming and then run NatNet again.
```

Comment:
In order to use code snippets inside alerts in nubook there needs to be a newline afterwards - I know there's none in this one but just for best practice
```suggestion
<Alert type='info'>

NatNet may not always pick up motive packets immediately. This may be dependent of the amount of traffic on your local network or how it is configured. Best Practice to ensure the least flakeyness is to start streaming packets on motive significantly before you run NatNet and if NatNet still doesn't work, try stopping and starting NatNet a few times. If this still doesn't work, try restarting motive streaming and then run NatNet again.
```